addons/l10n_ke_edi_oscu/__init__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+#
+from . import models
+from . import wizard
+from . import utils
+from . import const
+
+_logger = logging.getLogger(__name__)
+
+
+def _post_init_hook(env):
+    # UNSPSC category codes can be used in Kenya.
+    product_unspsc = env['product.unspsc.code'].search([('active', '=', False), ('code', '=ilike', '%00')])
```

Comment:
I don't think it would cause issues for companies in other countries. As far as I can see, the UNSPSC codes are just needed for some EDIs. I assume that if an EDI doesn't accept a Category Code for a product, this would simply result in some kind of error message for the user, who would then need to choose a different code.